connect/src/com/telenor/connect/ConnectSdk.java

```diff
@@ -306,6 +308,19 @@ public static synchronized void sdkInitialize(Context context) {
                 sRedirectUri);
     }
 
+    public static String getMccMncLoginHint() {
```

Comment:
Done. I was experimenting a bit with the workflow here and just amended the last commit (which is not you should do in GitHub :) ) , so you won't be able to see the prev commit and the diff.